uber/templates/preregistration/confirm.html

```diff
@@ -58,12 +58,14 @@
     <td></td>
     <td> <input type="submit" value="{% if attendee.placeholder %}Register{% else %}Update My Info{% endif %}" id="updateButton"/> </td>
 </tr>
```

Comment:
I swear we already had this if statement in place.  Must have gotten deleted somehow.